### ISSUE TITLE
question about the loss 

### ISSUE THREAD

**Issue by eternal-heart** [phermosilla/TotalDenoising#5]
Hello,I have traned the model and test the model with the Ruedataset.In the ''log'' documention,I see that the loss increases with the increase of iteration.This is my question.Shouldn't the loss get smaller and smaller?Why it increase?

**Comment by eternal-heart** [comment]
Can you explain the meaning of the lossOrder "order of the Lk loss function".

**Comment by ia-usami** [comment]
This is suck. No training/testing set release. No pretrain weights release. If you try to train it on your own data, you have no idea why the model is not converging. You just can not make it works. This is an incomplete open source.

**Comment by phermosilla** [comment]
Hello eternal-heart,

The increase of the loss is normal if you use the L0 loss. The order of the Lk loss function is the exponent of the loss. If you use L2 the loss is (pred-gt)^2, the L1 is (pred-gt)^1 and the L0 is (pred-gt)^0. Since the L0 cannot be used like this, the exponent is decreased linearly from 2 to 0 during training which leads to the increase of the loss value you see in the plots. This is normal behavior and your model should be trained in the last epoch. You can read more about this loss in the paper and also in the Noise2Noise paper where also proof is provided that L0 converges to the mode of a distribution.

Hello ia-usami,

As I explain here, this is the normal behavior of the loss. You should read our paper and the paper where this loss was introduced before complaining here.
